src/cljs/org/broadinstitute/firecloud_ui/page/workspace/monitor/tab.cljs

```diff
@@ -78,22 +82,27 @@
 
 (react/defc Page
   {:refresh
-   (fn [{:keys [props refs]}]
-     (let [nav-context (nav/parse-segment (:nav-context props))
+   (fn [{:keys [props refs]}] 
+     (let [nav-context (nav/parse-segment (:nav-context props)) 
+           workspace (:workspace props)
            selected-submission-id (not-empty (:segment nav-context))]
        (if selected-submission-id
          (nav/back nav-context)
          (react/call :reload (@refs "submissions-list")))))
    :render
    (fn [{:keys [props]}]
      (let [workspace-id (:workspace-id props)
+           workspace (:workspace props)
```

Comment:
This smells like a DRY violation to me. Does `workspace` have an ID? If so, what would the behavior be if that was different than the value in `workspace-id`?